new_core/state.js

```diff
@@ -4,8 +4,20 @@ export const state = {
 }
 
 export const input = {
```

Comment:
For keyboard, this should be okay. But, in the long term, dishing this off in an outside JSON file would be great for quickly editing these values.

Reply:
it could be separated when the dynamic file loader is merged/complete :)